static/js/containers/App.js

```diff
@@ -88,49 +85,19 @@ class App extends React.Component {
       location: { pathname },
       dispatch,
     } = this.props;
-    let [ complete, info ] = validateProfileComplete(profile);
+    let [ complete, url, errors] = validateProfileComplete(profile);
     if (
       userProfile.getStatus === FETCH_SUCCESS &&
       profile.agreed_to_terms_of_service &&
       !PROFILE_REGEX.test(pathname) &&
       !complete
     ) {
-      const { url, title, text } = info;
-      dispatch(updateDialogText(text));
-      dispatch(updateDialogTitle(title));
-      dispatch(setDialogVisibility(true));
+      dispatch(startProfileEdit);
```

Comment:
not quite the same issue, with this change (`startProfileEdit` -> `startProfileEdit()`) I do actually see validation errors (before they weren't showing up at all).

this is a different issue than the profile PATCH promise async issue.